schemas/prototypeApplication.json

```diff
@@ -3,654 +3,12507 @@
+      "additionalProperties": false,
+      "properties": {
+        "area": {
+          "$ref": "#/definitions/Area"
+        },
+        "site": {
+          "$ref": "#/definitions/GeoJSON"
+        }
+      },
+      "required": [
+        "site",
+        "area"
+      ],
+      "type": "object"
+    },
+    "GeoBoundaryPrototype": {
+      "additionalProperties": false,
+      "properties": {
+        "area": {
+          "$ref": "#/definitions/Area"
+        },
+        "site": {
+          "$ref": "#/definitions/GeoJSON"
+        }
+      },
+      "required": [
+        "site",
+        "area"
+      ],
+      "type": "object"
+    },
+    "GeoJSON": {
+      "anyOf": [
+        {
+          "$ref": "#/definitions/Geometry"
+        },
+        {
+          "$ref": "#/definitions/Feature"
+        },
+        {
+          "$ref": "#/definitions/FeatureCollection"
+        }
+      ],
+      "description": "Union of GeoJSON objects."
+    },
+    "GeoJsonProperties": {
+      "anyOf": [
+        {
+          "type": "object"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "Geometry": {
+      "anyOf": [
+        {
+          "$ref": "#/definitions/Point"
+        },
+        {
+          "$ref": "#/definitions/MultiPoint"
+        },
+        {
+          "$ref": "#/definitions/LineString"
+        },
+        {
+          "$ref": "#/definitions/MultiLineString"
+        },
+        {
+          "$ref": "#/definitions/Polygon"
+        },
+        {
+          "$ref": "#/definitions/MultiPolygon"
+        },
+        {
+          "$ref": "#/definitions/GeometryCollection"
+        }
+      ],
+      "description": "Geometry object. https://tools.ietf.org/html/rfc7946#section-3"
+    },
+    "GeometryCollection": {
+      "additionalProperties": false,
+      "description": "Geometry Collection https://tools.ietf.org/html/rfc7946#section-3.1.8",
+      "properties": {
+        "bbox": {
+          "$ref": "#/definitions/BBox",
+          "description": "Bounding box of the coordinate range of the object's Geometries, Features, or Feature Collections. The value of the bbox member is an array of length 2*n where n is the number of dimensions represented in the contained geometries, with all axes of the most southwesterly point followed by all axes of the more northeasterly point. The axes order of a bbox follows the axes order of geometries. https://tools.ietf.org/html/rfc7946#section-5"
+        },
+        "geometries": {
+          "items": {
+            "$ref": "#/definitions/Geometry"
+          },
+          "type": "array"
+        },
+        "type": {
+          "const": "GeometryCollection",
+          "description": "Specifies the type of GeoJSON object.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "geometries",
+        "type"
+      ],
+      "type": "object"
+    },
+    "Integer": {
+      "format": "integer",
+      "type": "number"
+    },
```

Comment:
This is a first attempt at adding a specific "Integer" type via utils defintions - but don't think this is quite what we want yet?

Reply:
Okay further investigation here: 
- our lib `ts-json-schema-generator` does NOT support `integer` types (yet?): https://github.com/vega/ts-json-schema-generator?tab=readme-ov-file#current-state
- alternatively `typescript-json-schema` does support with an annotation like this: https://github.com/YousefED/typescript-json-schema?tab=readme-ov-file#annotations (this is a good example of our goal state)

Is it insane to think about switching TS generation libs? This thread has a lot of comparison notes, opinions: https://github.com/vega/ts-json-schema-generator/issues/101 (one for a followup PR for sure!)